src/PointwiseFunctions/AnalyticData/NewtonianEuler/KhInstability.hpp

```diff
@@ -58,19 +59,39 @@ namespace AnalyticData {
  * is its horizontal bimedian. The initial pressure is set equal to a constant,
  * and the system is evolved assuming an ideal fluid of known adiabatic index.
  * Finally, in order to excite the instability, the vertical velocity is
- * initialized to a small perturbation
+ * initialized to
  *
  * \f{align*}
  * v_y(x, y) = A\sin(4\pi x)
  * \left[\exp\left(-\dfrac{(y - y_\text{top})^2}{2\sigma^2}\right) +
  * \exp\left(-\dfrac{(y - y_\text{bot})^2}{2\sigma^2}\right)\right],
  * \f}
  *
- * where \f$A\f$ is the amplitude, \f$\sigma\f$ is a characteristic length for
- * the perturbation width, and \f$y_\text{top} = y_\text{mid} + b/2\f$ and
+ * whose net effect is to perturb the horizontal boundaries of the strip
+ * periodically along the \f$x-\f$axis. Here \f$A\f$ is the amplitude,
+ * \f$\sigma\f$ is a characteristic length for the perturbation width,
+ * and \f$y_\text{top} = y_\text{mid} + b/2\f$ and
  * \f$y_\text{bot} = y_\text{mid} - b/2\f$ are the vertical coordinates
  * of the top and bottom boundaries of the strip, respectively.
  *
+ * \note The periodic form of the perturbation enforces the horizontal
+ * extent of the domain to be a multiple of 0.5. The domain chosen in
+ * \cite Schaal2015 is the square \f$[0, 1]^2\f$, so the perturbation covers
+ * two wavelengths. In addition, the authors ran their simulations using
```

Comment:
Since the wavelength is fixed and the domain size is not, I would rephrase this as: "..., so the perturbation covers two wavelengths." -> "..., which covers two wavelengths of the perturbation."

Reply:
Much better, yes. Thanks!